components/admin_console/reset_email_modal/reset_email_modal.tsx

```diff
@@ -33,11 +40,23 @@ export default class ResetEmailModal extends React.PureComponent {
         this.emailRef = React.createRef();
     }
 
-    doSubmit = (e) => {
+    public componentWillUnmount(): void {
```

Comment:
What's this `componentWillUnmount` needed for? I don't think we should need to clear any state when the component is unmounted.

Reply:
Hi, tahnk you for your feedback  @hmhealey . 
 Yes, you're right, I already modify the code.